src/rpc/mempool.cpp

```diff
@@ -669,6 +670,136 @@ static RPCHelpMan savemempool()
     };
 }
 
+static RPCHelpMan submitpackage()
+{
+    return RPCHelpMan{"submitpackage",
+        "\nSubmit a package of raw transactions (serialized, hex-encoded) to local node (-regtest only).\n"
```

Comment:
Seems like this PR should just be a modification to sendrawtransaction

Reply:
sendrawtransaction is not regtest-only

Reply:
Nor should this be, IMO. Certainly not long-term.

But regardless, the regtest check can be part of the existing RPC too. (I don't think it'd be the first case of such either IIRC)